File: apps/backend-java/hrm-service/src/main/java/io/arda/hrm/domain/Customer.java
package io.arda.hrm.domain;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;
import java.util.UUID;

@Table("customers")
public record Customer(
    @Id UUID id,
    String customerCode,
    String name,
    String status, // PENDING, ACTIVE, REJECTED
    UUID cccdFileId
) {}

File: apps/backend-java/crm-service/src/main/kotlin/io/arda/crm/domain/Customer.kt
package io.arda.crm.domain

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table
import java.util.UUID

@Table("customers")
data class Customer(
    @Id val id: UUID? = null,
    val customerCode: String,
    val name: String,
    val status: String, // PENDING, ACTIVE, REJECTED
    val cccdFileId: UUID?
)
